prime-router/docs/proposals/0009-internal-tools-migration-to-superset.md

```diff
@@ -0,0 +1,50 @@
+## Background
+
+The PRIME ReportStream uses Metabase for business analytics. Metabase is a business intelligence tool that allows for the exploration, investigation, and visualization of data. With this internal tool, our team is able to generate charts and dashboards from our stored data. Even though this instrument provides quality charts, it also has its shortcomings:
+
+### Limitations 
+
+- Query complexity 
+    - Limited to simple queries only
+    
+- Data Source 
+    - Only works well with a single SQL data source
+    - Does not allow for joining of data
+    - Unable to upload CSV file format
+    
+- Formatting
+    - Can be difficult to customize charts
+    
+
+As our data input becomes larger and keeping scalability in mind, these limitations will become problematic. It will be more time-consuming to write complex queries and to customize charts than it is now.
+
+## Goals
+
+Our goal is to meet the demands of scalability while maintaining the ability to analyze data by dicing and visualization of charts and reports. Other key factors to consider are:
+
+### Key Factors
+
+- Open Source
+- Works with Azure
+- Has more visualization options (chart types)
+- Can connect to multiple data sources
+- Able to run complex queries (joins)
+- Low learning curve
+- Accepts CSV format
+- capable of running locally (for testing)
+- Highly customizable
```

Comment:
I'm sure there are others, but I can think of two requirements that Metabase does not meet:
1.  Ability to provide canned dashboards to our customers, senders, and receivers.    This would likely involve integration with Okta to provide SSO, as well as ways to ensure that we can securely stovepipe data per organization.
2. Maps that include US territories.   Bonus if it can map by zip code, and county.

Can superset meet these reqs?

Reply:
>     1. Ability to provide canned dashboards to our customers, senders, and receivers.    This would likely involve integration with Okta to provide SSO, as well as ways to ensure that we can securely stovepipe data per organization.

Canned dashboards for sure; using external auth is not something that I know of the top off my head. To be investigated!

>     2. Maps that include US territories.   Bonus if it can map by zip code, and county.

Superset has easy visualization mechanisms; mapping is one of them. Here's a gallery of some of the built-in visualizations: https://superset.apache.org/gallery